lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -96,7 +96,7 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
      */
     protected function doContains($id)
     {
-        return $this->redis->exists($id);
+        return $this->redis->exists($id) > 0 ? true : false;
```

Comment:
`$this->redis->exists($id) > 0` would be enough.

Since it previously returned bool, `bool > integer` is invalid, this needs to be rewritten properly. I.e.:
```
$result = $this->redis->exists($id);

if (is_bool($result)) {
    return $result;
}

return $result > 0;
```

Before anyone says _it's a function call, it's slow!_ - [it won't be](https://github.com/php/php-src/blob/d0ee2a8254c69546c7191ac26cdc0499eba27acf/ext/standard/type.c#L232).

Reply:
You need to use `\is_bool` for opcache optimizations to take effect.

Reply:
`use function is_bool`, but that is enforced by our CS so I didn't even mention it. :)

Reply:
@Tobion @Majkl578 Should I switch to `\is_bool` or keep the latest committed change ?